CMakeLists.txt

```diff
@@ -19,37 +23,60 @@ find_package(tf2_geometry_msgs REQUIRED)
 find_package(tf2_ros REQUIRED)
 find_package(Eigen3 REQUIRED)
 
+set(includes
+  include
+  ${EIGEN3_INCLUDE_DIRS}
+  ${rclcpp_INCLUDE_DIRS}
+  ${geographic_msgs_INCLUDE_DIRS}
+  ${geometry_msgs_INCLUDE_DIRS}
+  ${diagnostic_msgs_INCLUDE_DIRS}
+  ${diagnostic_updater_INCLUDE_DIRS}
+  ${nav_msgs_INCLUDE_DIRS}
+  ${sensor_msgs_INCLUDE_DIRS}
+  ${tf2_ros_INCLUDE_DIRS}
+  ${tf2_INCLUDE_DIRS}
+  ${tf2_geometry_msgs_INCLUDE_DIRS}
+)
+
+set(libraries
+  rl_lib
```

Comment:
As well here -- you're getting closer and closer on the current ROS2 cmake practices. I know it was a pain in earlier releases, but its pretty streamlined now.